torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14102,10 +14102,12 @@ def ref_pairwise_distance(input1, input2):
                     sample_inputs_func=sample_inputs_zeta),
     OpInfo('logsumexp',
```

Comment:
This won't test the functionality you're adding. The PR needs to extend the sample inputs to include samples that set `b` and `return_sign`. A reference operation to `scipy.logsumexp` should be added to the OpInfo, too, and additional testing may be needed if the behavior is very complicated.

Reply:
I think that should be done, including the reference implementation.